Makefile.am

```diff
@@ -235,10 +236,12 @@ ci-installcheck: ci-prepare installcheck
 	killall -HUP pkcsslotd || true
 	@sbindir@/pkcsslotd
 	cd ${srcdir}/testcases && export PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so && export PKCS11_USER_PIN=$(PKCS11_USER_PIN) && ./misc_tests/p11sak_test.sh | tee log-p11sak.txt
+	@sbindir@/pkcsstats
```

Comment:
Doesn't this only print the statistics for the current user?  The CI runs the tests under su to a special user `tester`.  Don't we then need to add the `--all` flag?

Reply:
good catch. Yes, lets use --all (and --delete-all and --reset-all at the other places).